pyproject.toml

```diff
@@ -64,6 +64,7 @@ run = {branch = true, parallel = true, source = [
 paths.source = [
     "src",
     ".tox*/*/lib/python*/site-packages",
+    ".tox*\\*\\Lib\\site-packages",
```

Comment:
https://github.com/tox-dev/tox/blob/main/pyproject.toml

Reply:
ah that's good to know, thanks!